### PR TITLE
[lit-next] Switch to nicer rollup file size summary plugin

### PR DESCRIPTION
**Pull request by aomarks**
image

image

### CODE REVIEW

**Review by justinfagnani — APPROVED** [review]

File: packages/lit-element/rollup.config.js [hunk 1]
```diff
@@ -112,10 +112,7 @@ export default {
         },
       },
     }),
-    filesize({
-      showMinifiedSize: false,
-      showBrotliSize: true,
-    }),
+    summary(),
```
[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-element/rollup.config.js:115]: It didn't do anything when I tried it

[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-element/rollup.config.js:115]: Yeah, it just doesn't do brotli: https://github.com/yousifalraheem/rollup-plugin-summary/blob/master/index.js#L71

[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-element/rollup.config.js:115]: filed https://github.com/yousifalraheem/rollup-plugin-summary/issues/11

[the remark this answers is not in the document]
Comment (justinfagnani) [reply packages/lit-element/rollup.config.js:115]: ah, I must have been reading the wrong docs

[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-element/rollup.config.js:115]: And today @yousifalraheem has already released a new version with Brotli support, plus a fix for `--watch` mode! Awesome, thank you!

**Pull request merged**